pyproject.toml

```
@@ -7,3 +7,51 @@ disable_error_code = ["union-attr", "annotation-unchecked", "import-untyped"]
 explicit_package_bases = true
 warn_unused_ignores = true
 exclude = ['tests']
+
+[tool.poetry]
+name = "tagstudio"
+version = "9.1.1"
+packages = [{include = "tagstudio/tag_studio.py"}, {include = "tagstudio/src/core"}, {include = "tagstudio/src/qt"}, {include = "tagstudio/resources"}, {include = "tagstudio/__init__.py"}]
```

can this be put in multiline for better readability?
```suggestion
packages = [
    {include = "tagstudio/tag_studio.py"},
    {include = "tagstudio/src/core"},
    {include = "tagstudio/src/qt"},
    {include = "tagstudio/resources"},
    {include = "tagstudio/__init__.py"}
]
```

---

Sure